usecase/event/msg_revoke_test.go

```diff
@@ -0,0 +1,81 @@
+package event_test
+
+import (
+	event_entity "github.com/crypto-com/chain-indexing/entity/event"
+	"github.com/crypto-com/chain-indexing/usecase/model"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	event_usecase "github.com/crypto-com/chain-indexing/usecase/event"
+)
+
+var _ = Describe("Event", func() {
+	registry := event_entity.NewRegistry()
+	event_usecase.RegisterEvents(registry)
+
+	Describe("En/DecodeMsgRevoke", func() {
+		It("should able to encode and decode to the same event", func() {
+			anyHeight := int64(1000)
+			anyTxHash := "4936522F7391D425F2A93AD47576F8AEC3947DC907113BE8A2FBCFF8E9F2A416"
+			anyMsgIndex := 2
+
+			anyParams := model.MsgRevokeParams{}
```

Comment:
Sorry I didn't notice this yesterday 🙏 

I think here we should set some test values inside the `model.MsgRevokeParams`.

For example, here is a reference of how other event's test case works.
https://github.com/crypto-com/chain-indexing/blob/master/usecase/event/msg_ibc_channel_open_ack_test.go

Similar cases in other `usecase/event/` test cases as well 🙏

Reply:
may i know the purpose of `must.Do(decoder.Decode(anyRawValue))`?

Reply:
May i know the reason to do the raw json decoder in those test cases rather than direct assignment? i think they have no dependency to the event decoder.

e.g.
```
var anyRawValue map[string]interface{}
			var anyRawMsgChannelOpenAck ibc_model.RawMsgChannelOpenAck
			decoder, _ := mapstructure.NewDecoder(&mapstructure.DecoderConfig{
				WeaklyTypedInput: true,
				DecodeHook: mapstructure.ComposeDecodeHookFunc(
					mapstructure.StringToTimeDurationHookFunc(),
					mapstructure.StringToTimeHookFunc(time.RFC3339),
					ibcmsg.StringToDurationHookFunc(),
					ibcmsg.StringToByteSliceHookFunc(),
				),
				Result: &anyRawMsgChannelOpenAck,
			})
			json.MustUnmarshalFromString(`
{
  "@type": "/ibc.core.channel.v1.MsgChannelOpenAck",
  "port_id": "transfer",
  "channel_id": "channel-0",
  "counterparty_channel_id": "channel-0",
  "counterparty_version": "ics20-1",
  "proof_try": "CqcCCqQCCi1jaGFubmVsRW5kcy9wb3J0cy90cmFuc2Zlci9jaGFubmVscy9jaGFubmVsLTASMggCEAEaFQoIdHJhbnNmZXISCWNoYW5uZWwtMCIMY29ubmVjdGlvbi0wKgdpY3MyMC0xGgsIARgBIAEqAwACICIrCAESBAIEICAaISCKn2oMLG+m7x551nh06PSUqF3nF6zuDAUZzHWN3Y7R6iIrCAESBAQGICAaISAl+qVfRZQA8J2FNbAhThANLKQJekTVQlahuZE83dRMGyIrCAESBAgQICAaISAnLE8YAY/vNnafNqIgxC5dnti4svEmGWT97i5E1czeVSIrCAESBAokICAaISD6l2Xp3knzZRXOdJhJ8ToqY0qLt6QgDUruuoeIyeXg6wrTAQrQAQoDaWJjEiDpdjgQlYa7NrP+ylMayPUCN/n48pTXiof8JywRXiIjxxoJCAEYASABKgEAIiUIARIhAUayD1sqD5N3koJLoD5Cenp4hU5w5Hs10qmKy+U4lvkTIiUIARIhAaj29iVS1fIRuqzuZBN/XoI11+IAJTrJPqBHC1ZQFHsbIiUIARIhAdmJCKPzDC4kc5/SoZhd1InuuJbVmFMLbq5ugt5dhByhIicIARIBARogEqy+GTuqKMTr9SDspw2rkC/9N0sLQcUcY4c9336zn3M=",
  "proof_height": {
    "revision_number": "2",
    "revision_height": "17"
  },
  "signer": "cro10snhlvkpuc4xhq82uyg5ex2eezmmf5ed5tmqsv"
}
`, &anyRawValue)
			must.Do(decoder.Decode(anyRawValue))
```

Reply:
I think it is for using real sample data for the unit test. 

If we use direct assignment, it has a possibility of failure when running with real data, but if we test it with real data, we could be more confident before deploying it.

Reply:
Umm, but they have no dependency on the event encoder and decoder i.e. this part is independent from the implementation

Reply:
Sorry, I updated the previous comments 😂 

I got what you mean now and I agree with you!